pkg/parser/phc2sys_parser.go

```diff
@@ -191,6 +192,7 @@ func extractRegularPhc2Sys(parsed *phc2sysParsed) (*Metrics, error) {
 	clockState := clockStateFromServo(parsed.ServoState)
 
 	return &Metrics{
+		ConfigName: parsed.ConfigName,
```

Comment:
metrics shoudl not have config name , since every time ptp config is reapplied the config index changes  Explain to me why you need this pls